### ISSUE TITLE
Vertical sharding  how to code for java API 

### ISSUE THREAD

**Issue by liruhui** [apache/shardingsphere#24704]
# 1. Official document
![image](https://user-images.githubusercontent.com/24687986/226504809-176900f4-aaa8-4f02-ae3d-7dd4aa87588a.png)

# 2.  My needs
![image](https://user-images.githubusercontent.com/24687986/226505288-a8b2899f-76dd-47f7-b93f-2cbd035c5f62.png)

# 3. javaCode 
![image](https://user-images.githubusercontent.com/24687986/226505831-de597d2a-ea6a-4597-9862-43c9ca5fc13b.png)

# 4. I do not  know  how to code correctly  please Tell me

**Comment by terrymanu** [comment]
Could you change the title for search friendly?

**Comment by liruhui** [comment]
> Could you change the title for search friendly? The title does not make sense.

title changed  but  The problem has not been solved.

**Comment by liruhui** [comment]
@RaigorJiang  
> 1. If the table already exists in a different DB, no configuration is required, and ShardingSphere can route correctly;
> 2. If you want to execute CREATE TABLE through ShardingSphere, you can only set data nodes in the sharding table rule like the configuration you provided.

```
vertical sharding database  not  need  any configuration   thit is OK  , 
but  vertical sharding table no configuration is error 

for example 

one table has three column (id, name age) , ->vertical sharding table->  table_1 has column(id,name)， table_2 has 
colume(id,age)    columes named  "name" and “age” in different table   maybe table name is  different
``` 

```java
    public static void main(String[] args) throws Exception{
        Map<String, DataSource> mapDatasource = new HashMap<>();
        HikariDataSource table1Database = new HikariDataSource();
        table1Database.setDriverClassName();
        table1Database.setJdbcUrl();
        table1Database.setUsername();
        table1Database.setPassword();

        HikariDataSource table2Database = new HikariDataSource();
        table2Database.setDriverClassName();
        table2Database.setJdbcUrl();
        table2Database.setUsername();
        table2Database.setPassword();
        mapDatasource.put("table1Database",table1Database);
        mapDatasource.put("table2Database",table2Database);

        DataSource dataSource = ShardingSphereDataSourceFactory.createDataSource(mapDatasource, null, null);
        Connection connection = dataSource.getConnection();
        connection.prepareStatement("insert into table(id,name,age) values(1,'aaaa',22)").execute();
        connection.close();
    }
```

![image](https://user-images.githubusercontent.com/24687986/229264593-c13e6ee8-fc4a-4f79-8765-a34cb4251bca.png)


```
I  think  this need configuraiton, but I not found API in RuleConfiration(Interface and implements class )   I do not know  how to config
```

**Comment by RaigorJiang** [comment]
Closed for #25078.
